tox.ini

```diff
@@ -75,6 +75,8 @@ basepython = python3.11
 deps =
     wagtail>=5.2
 
+    psycopg2>=2.9
```

Comment:
**Question:** Could someone run the local environment with SQLite instead? I guess this requires having `pg_config` installed locally. For users who don't want to use Postgres, this kind of means they would get an error when this is being installed.

Does this need a full-on explanation in the documentation? Or separate interactive environments?

I guess the question is how high we want the contribution barrier to be, e.g. is a baseline that someone will be OK using Postgres and has packages already installed locally?